framework/moose.mk

```diff
@@ -11,6 +11,10 @@ moose_SRC_DIRS += $(FRAMEWORK_DIR)/contrib/mtwist
 moose_SRC_DIRS += $(FRAMEWORK_DIR)/contrib/jsoncpp
 moose_SRC_DIRS += $(FRAMEWORK_DIR)/contrib/pugixml
 
+.PHONY: verify_conda_libmesh
+verify_conda_libmesh:
```

Comment:
didn't you have to add that target to `all`?

Reply:
That's right, whoops.